### ISSUE TITLE
Direct Logit Attribution not replicating

### ISSUE THREAD

**Issue by soniajoseph** [soniajoseph/ViT-Prisma#72]
Direct Logit Attribution not replicating

**Context**
I am replicating the [Exploratory Analysis notebook](https://colab.research.google.com/github/neelnanda-io/TransformerLens/blob/main/demos/Exploratory_Analysis_Demo.ipynb#scrollTo=M7OOwrd2V4L9), which goes through Reverse Engineering GPT-2 Circuit and also basic TransformerLens functionality.

I am replicating the functionality [in this notebook here ](https://github.com/soniajoseph/prisma-development/blob/main/prisma-development/exploratory-analysis/Exploratory%20Analysis.ipynb)with localizing cat/non-cat in the net as a test case.
So far, the replication is acting as a good test of our functionality.

I am not able to replicate the “average logit difference.” In my notebook, this is under the "Direct Logit Attribution" section of both notebooks.  Projecting the intermediate activations onto the output head is supposed to equal taking the logits. However, I am getting a mean of 9.21 for the first method and  7.31  for the taking the logits directly.

**What could be going wrong:**
1. Layer norm folding, centering the weight matrices, or centering the unembed may have bugs or may be incorrectly implemented. One step is checking each of these operations in the unit test.
2. There may be a bug in my implementation in that notebook

**Request**
Double-check my work and find the bug (I have been staring at this for awhile and extra eyes would be immensely helpful! Without getting this notebook to work, we cannot be sure that the basic functionality is set up properly)

**Comment by soniajoseph** [comment]
Solved by @themachinefan in the commit above!
